trace-utils/src/send_data/mod.rs

```diff
@@ -473,18 +497,30 @@ mod tests {
         construct_tracer_payload(vec![chunk], header_tags, root_tags)
     }
 
-    fn compute_payload_len(payload: &[TracerPayload]) -> usize {
-        let agent_payload = construct_agent_payload(payload.to_vec());
-        let serialized_trace_payload = serialize_proto_payload(&agent_payload).unwrap();
-        serialized_trace_payload.len()
+    fn compute_payload_len(collection: &TracerPayloadCollection) -> usize {
+        match collection {
+            TracerPayloadCollection::V07(payloads) => {
+                let agent_payload = construct_agent_payload(payloads.to_vec());
+                let serialized_trace_payload = serialize_proto_payload(&agent_payload).unwrap();
+                serialized_trace_payload.len()
+            }
+            _ => 0,
+        }
     }
 
-    fn rmp_compute_payload_len(payload: &Vec<TracerPayload>) -> usize {
-        let mut total: usize = 0;
-        for payload in payload {
-            total += rmp_serde::to_vec_named(payload).unwrap().len();
+    fn rmp_compute_payload_len(collection: &TracerPayloadCollection) -> usize {
```

Comment:
what `rmp` means?

Reply:
rmp stands for rust messagepack (https://docs.rs/rmp-serde/latest/rmp_serde/index.html)